pom.xml

```diff
@@ -548,67 +547,6 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
-        <groupId>pl.project13.maven</groupId>
-        <artifactId>git-commit-id-plugin</artifactId>
```

Comment:
This plugin generates a `.git` file that contains the exact commit id, message, and other info that helps to find out the information about a specific build, especially when it was done for the master. Instead of removing the file at all, you can configure the plugin to include only entries that will be the same for different environments.

Reply:
See https://github.com/git-commit-id/git-commit-id-maven-plugin/blob/master/docs/using-the-plugin.md for detailed plugin usage.

Reply:
@hboutemy Do you think you'll be able to address @vvysotskyi's comment?

Reply:
I have one question: how do you handle official Apache release building done from source tarball instead of Git?

I imagine this plugin can be useful for SNAPSHOTs, where you don't know precisely which commit is built

Reply:
> I have one question: how do you handle official Apache release building done from source tarball instead of Git?
> 
> I imagine this plugin can be useful for SNAPSHOTs, where you don't know precisely which commit is built

@jnturton Maybe you can take this one?

Reply:
@hboutemy our official Apache builds are created from a Git commit using Maven release plugins, not from a source tarball (although this is an output of the process)...

Reply:
yes, that's expected to be done like this while releasing: no choice, the beginning of the release process is a source control. It has been cvs, then svn, now git, perhaps something else in the future.
the fact is that when voting, the source tarball is what is registered at Apache level, with distribution area, because it's the same format for years and for foreseeable future
and sadly, if you have such dependency on Git, you can't rebuild from that official source tarball and get the same output

that's why I propose to get the best of the 2 worlds:
- when building a SNAPSHOT, yes, use that Git binding because it gives you useful details
- when building a release, just don't use that Git binding because it causes that issue

Reply:
@hboutemy would it be sufficient, as @vvysotskyi proposed, for us to remove from the generated git.properties file those of the plugin's outputs that are not are not reproducible.  I.e. kill lines such as these?


```
│ -git.build.host=jtpc
│ -git.build.time=22.02.2022 @ 20\:20\:27 SAST
│ +git.build.host=c23e344f8445
│ +git.build.time=26.02.2022 @ 17\:58\:09 UTC
│  git.build.user.email=
│  git.build.user.name=
```

In general, I'm a fan of the idea of being to build from only a source tarball with no dependency on the VCS _de jour_ but on the other hand, adjusting this git-commit-id plugin looks less disruptive?

Reply:
ok, let's do like this :)